multipool-tracker/src/multipool_storage/errors.rs

```diff
@@ -0,0 +1,20 @@
+use ethers::types::Address;
+
+pub enum MultipoolErrors {
+    SpecificTypeNotFound,
+    FieldMissing,
+    AssetNotFound(Address),
+    PriceNotFound(Address),
+    QuantityNotFound(Address),
+    OperationsErrors,
+    AdditionOverflow,
+    MultiplicationOverflow,
+    ShiftOverflow,
+}
```

Comment:
The format I slightly suggest to take as base:
```rust
pub enum MultipoolErrors {
   QuantitySlotMissing(Address),
   PriceMissing(Address),
   ShareMissing(Address),
   Overflow(MultipoolOverflowErrors),
   ...
   // Any multipool errors except the ones that are connected with overflow (data doesn't suit into value capacity)  
}

pub enum MultipoolOverflowErrors {
   QuotedQuantityOverflow, // Overflows that are attached to what we get as result of operation and what overflows
   ...
}
```

Reply:
Error always should describe things that  occurred in a way consumer of the error can understand it (e.g shift overflow doesn't describe anything to guy who executes any function. they shouldn't know wtf is happening under the hood, they need to know something like quote price overflow or something like this that they know might happen)